src/test/scala/uk/gov/hmrc/play/bootstrap/http/DefaultHttpClientSpec.scala

```diff
@@ -20,18 +20,18 @@ import com.github.tomakehurst.wiremock.client.WireMock._
 import org.joda.time.LocalDate
 import org.scalatest.TestData
 import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
-import org.scalatestplus.play.OneAppPerTest
+import org.scalatestplus.play.{OneAppPerSuite, OneAppPerTest, WsScalaTestClient}
 import play.api.Application
 import play.api.inject.bind
 import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.ws.WSClient
 import uk.gov.hmrc.http._
 import uk.gov.hmrc.play.audit.http.connector.AuditConnector
 import uk.gov.hmrc.play.bootstrap.http.utils._
 import uk.gov.hmrc.play.test.UnitSpec
 
-class DefaultHttpClientSpec extends UnitSpec with ScalaFutures with IntegrationPatience with WiremockTestServer with OneAppPerTest {
-
-  override def newAppForTest(testData: TestData): Application = new GuiceApplicationBuilder()
```

Comment:
Could you talk us through these changes here? Are they related to the underlying issue or some cleaning up?

Reply:
Moved to OneAppPerSuite Rather than OneAppPerTest due to the WSClient in Play being closed after the first test and not restarting each time. I first tried to move it to a later scalatestplus version to move to GuiceOneAppPerTest which has been used elsewhere and seemed to work. However this caused a ripple of changes that broke most of the other tests. So settled moving to Suite, as I believed the tests were focused on functionality without any crossover.

Reply:
Ok cool, that makes sense. Thanks!

Reply:
@Tobias1087 you can achieve the same thing by changing 
`lazy val myHttpClient` to `def myHttpClient`